ext/posix/tests/posix_setegid_basic.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Test function posix_setegid() by calling it with its expected arguments
+--SKIPIF--
+<?php 
+    if(!extension_loaded("posix")) die("skip - POSIX extension not loaded"); 
+?>
+--FILE--
+<?php
+var_dump(posix_setegid(posix_getegid()));
```

Comment:
@shaikhul yes, right, I forgot about sudo stuff. then your one should be fine. Thanks!